CHANGELOG.md

```diff
@@ -4,37 +4,33 @@
 
 [Full Changelog](https://github.com/stellargraph/stellargraph/compare/HEAD...v0.8.2)
 
-**New algorithms:**
+### Major features and improvements
 
 - Cluster-GCN algorithm (an extension of GCN that can be trained using SGD) + demo [\#487](https://github.com/stellargraph/stellargraph/issues/487)
```

Comment:
I like the focus on major features, not just algorithms, but was wondering whether algorithms is important enough to have a sub-heading under this?

Reply:
I've added a nested list. What do you think?

Reply:
looks good, thanks!